src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(Lottery.getRandomBall());
+        }
+
     }
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
    }
```